pkg/gui/custom_patch_options_panel.go

```diff
@@ -28,6 +29,11 @@ func (gui *Gui) handleCreatePatchOptionsMenu() error {
 			OnPress: func() error { return gui.handleApplyPatch(true) },
 			Key:     'r',
 		},
+		{
+			Label:   "Copy patch to clipboard",
+			OnPress: func() error { return gui.copyPatchToClipbaord() },
+			Key:     gocui.KeyCtrlO,
```

Comment:
I propose `y`, since we're already in a menu, holding `ctrl` really isn't necessary, we have keyboard real-estate to use :)

And cuz `vim`.

Reply:
`vim` niiiice

Reply:
oh also, it looks like clipboard has a typo here :eyes:

Reply:
@mark2185  thanks for review fixed this.

@bashbunni  thank a lot for finding out the typo.